StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1118ParameterMustNotSpanMultipleLines.cs

```diff
@@ -54,7 +57,7 @@ public class SA1118ParameterMustNotSpanMultipleLines : DiagnosticAnalyzer
         /// </summary>
         public const string DiagnosticId = "SA1118";
         private const string Title = "Parameter must not span multiple lines";
-        private const string MessageFormat = "TODO: Message format";
+        private const string MessageFormat = "The parameter spans multiple lines. If the parameter is short, place the entire parameter on a single line. Otherwise, save the contents of the parameter in a temporary variable and pass the temporary variable as a parameter.";
```

Comment:
Do we really want a message this long? The Light bulb is cutting of parts of it
![Light bulb](https://cloud.githubusercontent.com/assets/1710209/7459791/bed7ed86-f29e-11e4-976e-1f63faf284d3.png)